homework4.4.py

```diff
@@ -0,0 +1,6 @@
+line = input('Enter any string: ')
+line2 = input('Enter any string: ')
+if line == line2:
+    print('Strings are identical')
+else:
+    print('Strings are not identical')
```

Comment:
This can be simplier
first = input("Enter first string: ")
second = input("Enter second string: ")

print(f"Is string {first} and string {second} the same? - {first == second}")